helm/loki/values.yaml

```diff
@@ -29,6 +29,10 @@ multiTenantAuth:
 
 imagePullSecrets: []
 
+giantswarm:
+  labelmonitoring:
```

Comment:
Why `labelmonitoring` instead of `monitoring` ?

Reply:
As opposed to the "normal" monitoring discovery which would be with servicemonitors rather than labels.

Reply:
Ok make sense